src/Corax/IndexEntry/IndexEntryWriter.cs

```diff
@@ -305,7 +305,7 @@ public void Write<TEnumerator>(int field, TEnumerator values, IndexEntryFieldTyp
         where TEnumerator : IReadOnlySpanIndexer
     {
         // We will include null values if there are nulls to be stored.           
-        int nullBitStreamSize = values.Length / (sizeof(byte) * 8) + values.Length % (sizeof(byte) * 8) == 0 ? 0 : 1;
+        int nullBitStreamSize = values.Length / (sizeof(byte) * 8) + (values.Length % (sizeof(byte) * 8) == 0 ? 0 : 1);
         byte* nullStream = stackalloc byte[nullBitStreamSize];
```

Comment:
If we'll set it as `Span<byte>`, that would also do range checks, and would spare us the error if it happened.

Reply:
But also introduce range checks in a performance sensitive piece of code. Tough choice. Let me see if I can evict somehow the range checks.

Reply:
Nope, range checks will happen on every bit flip.